cluster/gce/util.sh

```diff
@@ -2553,15 +2605,92 @@ function kube-up() {
     write-cluster-name
     write-controller-config
     create-autoscaler-config
-    create-master
-    create-nodes-firewall
-    create-nodes-template
-    if [[ "${KUBE_CREATE_NODES}" == "true" ]]; then
-      # Windows nodes take longer to boot and setup so create them first.
-      create-windows-nodes
-      create-linux-nodes
+    if [[ "${SCALEOUT_CLUSTER:-false}" == "true" ]]; then
```

Comment:
suggest to introduce a new function, say, scaleout-kube-up() and preserve the previous kube-up().

Reply:
this change requires admin cluster to be tp-tp mode which is not ideal for normal perf runs

Reply:
> this change requires admin cluster to be tp-tp mode which is not ideal for normal perf runs

This change support both scale-up and scale-out for kube-up.sh and will not block current perf runs

Reply:
> suggest to introduce a new function, say, scaleout-kube-up() and preserve the previous kube-up().

This is the quick version to support scale out in Kube-up.sh. we still need bunch of optimize for final solution.

Reply:
it might be an easier approach to do to perseve the kube-up(), instead of creating all the clusters in side this function, create scaleout clusters from outside of kube-up(). i.e.
1. each kube-up still start signle cluster
2. each kube-up will be controlled with flag to create slave nodes or not
3. the other logic, such as create shared stuff, pre-allocate IPs, etc remains as the same in the PR.